circus/tests/test_reloadconfig.py

```diff
@@ -168,5 +168,6 @@ def test_reload_ignorearbiterwatchers(self):
         statsd = a.get_watcher('circusd-stats')
         yield a.reload_from_config(_CONF['reload_statsd'])
         self.assertEqual(statsd, a.get_watcher('circusd-stats'))
+        yield self._tear_down_arbiter(a)
```

Comment:
I don't think actually fixed the tests, but I think that it did help stop having dangling plugin processes (as in https://github.com/circus-tent/circus/issues/1017#issuecomment-260566442 )